### ISSUE TITLE
"Static Assertion Checking" vs. "Static Verification"

### ISSUE THREAD

**Issue by samm82** [samm82/TestGen-Thesis#61]
In [deBoerEtAl2006](https://gitlab.cas.mcmaster.ca/crawfs1/TestGen-Thesis-Papers/-/blob/main/deBoerEtAl2006.pdf), "two complementary forms of assertion checking" are given: "runtime assertion checking" and "static verification" (ChalinEtAl2006, p. 343). This seems incorrect, especially since [LahiriEtAl2013](https://gitlab.cas.mcmaster.ca/crawfs1/TestGen-Thesis-Papers/-/blob/main/LahiriEtAl2013.pdf) uses the phrase "static assertion checking" (p. 345) (even though it's not formally defined). It seems like "static verification" should be replaced by "static assertion checking".

I just noted this discrepancy (see 5d2f109) and only included "static assertion checking" in my table since this made the most sense to me; should I do something else? Is it OK to be making these kinds of decisions at this point?

**Comment by JacquesCarette** [comment]
I feel like your first paragraph doesn't quite give me enough information (not without me chasing down a lot of information) to fully understand the issue.

From the best-guess-understanding, it does seem like you're doing the right thing.

**Comment by samm82** [comment]
Essentially, the source I linked to said that the "opposite" of "runtime assertion checking" is "static verification", but since there's a precedence for the term "static assertion checking", I think that's what they should have said (and is what I recorded).

This was more of a double check to see if I was doing something _obviously_ wrong. 😅 If you're unsure, I can provide more details, but honestly there might not be many (at least, many related to this specific "issue").
